content/en/getting_started/integrations/aws.md

```diff
@@ -0,0 +1,139 @@
+{{< img src="getting_started/integrations/cloudformation-options-1.png" alt="First part of CloudFormation options for creating Datadog stacks">}}
+
+3. Ensure that the the correct [Datadog site][9] parameter is used for the `DdSite` parameter to match the region of your Datadog account. Check the required boxes from AWS and click `Create stack`:
+{{< img src="getting_started/integrations/cloudformation-options-2.png" alt="Second part of CloudFormation options for creating Datadog stacks">}}
+
+
+This begins the creation process for the Datadog stack along with three nested stacks:
+{{< img src="getting_started/integrations/cloudformation-stacks-complete.png" alt="Datadog stacks completed in CloudFormation">}}
+
+4. From the integration tile on the Datadog side, enter the AWS account ID and AWS Role name. If the role name wasn't changed in the CloudFormation setup page, use `DatadogIntegrationRole`. Datadog automatically queries AWS with the provided values, and a successful query returns the following status:
+{{< img src="getting_started/integrations/integration-tile-status.png" alt="Datadog AWS integration tile with 'Account credentials are valid' message">}}
+
+5. Finally, click `Install integration`. For the purposes of this demonstration, all options have been selected (only `Collect custom metrics` is unticked by default):
+{{< img src="getting_started/integrations/integration-tile-complete.png" alt="Datadog AWS integration tile">}}
+
+It should be noted that there are multiple options within the integration tile to specify the data to be collected. This can be limited based on AWS service, namespace, or tags.
+With [EC2 automuting][10] enabled, Datadog uses host statuses reported by the CloudWatch API to distinguish between expected and unexpected instance terminations, preventing unwanted Monitor notifications.
+
+### Validation
+
+The automatically-provisioned [AWS overview dashboard][11] provides an immediate window into the metrics sent by your AWS services and infrastructure:
+{{< img src="getting_started/integrations/aws-dashboard.png" alt="AWS overview dashboard">}}
+
+
+## Sending Logs
+
+As a result of setting up the Datadog Forwarder function with the CloudFormation template, log collection can be configured from directly within the integration tile in Datadog. Click into the `Collect Logs` tab in the Integration tile and enter the ARN of the Forwarder lambda. With the ARN entered, click the check mark to establish the connection: 
+{{< img src="getting_started/integrations/integration-tile-collect-logs-1.png" alt="Datadog AWS integration tile 'Collect Logs' tab">}}
+
+Once completed, you'll be able to select from any of the log sources mentioned in [this guide][12] for sending AWS service logs. For this guide, we'll select only S3 access logs, and click `Save`:
```

Comment:
```suggestion
Once completed, you'll be able to select from any of the log sources mentioned in the [Send AWS services logs with the Datadog Lambda function guide][12] for sending AWS service logs. 

Select S3 access logs, and click `Save`:
```

Reply:
Once this is complete, you can select ...